docs/content/en/docs/implementing/evaluations.md

```diff
@@ -3,3 +3,85 @@ title: Evaluations
 description: Understand Keptn evaluations and how to use them
 weight: 150
 ---
+A `KeptnEvaluation` is a `KeptnMetric` that has a defined target value.
+A `KeptnEvaluation` resource is defined in a
+[KeptnEvaluationDefinition](../yaml-crd-ref/evaluationdefinition.md)
+yaml file.
+The example
+[app-pre-deploy-eval.yaml](https://github.com/keptn/lifecycle-toolkit/blob/main/examples/sample-app/version-3/app-pre-deploy-eval.yaml)
+file specifies the `app-pre-deploy-eval-2` evaluation as follows:
+
+```yaml
+apiVersion: lifecycle.keptn.sh/v1alpha3
+kind: KeptnEvaluationDefinition
+metadata:
+  name: app-pre-deploy-eval-2
+  namespace: podtato-kubectl
+spec:
+  objectives:
+    - keptnMetricRef:
+        name: available-cpus
+        namespace: podtato-kubectl
+      evaluationTarget: ">4"`
+```
+
+The `evaluationTarget` is set to be `>4`,
+so this evaluation ensures that more than 4 CPUs are available
+before the workload or application is deployed.
+
+The example
+[metric.yaml](https://github.com/keptn/lifecycle-toolkit/blob/main/examples/sample-app/base/metric.yaml)
+file defines the
+[KeptnMetric](../yaml-crd-ref/metric.md) resource
+that is named  `available-cpus`:
+
+```yaml
+apiVersion: metrics.keptn.sh/v1alpha3
+kind: KeptnMetric
+metadata:
+  name: available-cpus
+  namespace: podtato-kubectl
+spec:
+  provider:
+    name: my-provider
+  query: "sum(kube_node_status_capacity{resource='cpu'})"
+  fetchIntervalSeconds: 10
+```
+
+To integrate your `KeptnEvaluation` resource, you must:
+
+* Annotate your [Workloads](https://kubernetes.io/docs/concepts/workloads/)
```

Comment:
maybe we could rephrase this block reordering the sentences so that is easier to understand, what about: 

to run evaluation on your app you must: ...

to run evaluation on one of your workloads/daemosets/etc you must: ...

Reply:
That definitely reads better -- done!